tests/graph_panel/test.jsonnet

```diff
@@ -73,4 +73,7 @@ local graphPanel = grafana.graphPanel;
   alertsWithMultipleConditions: graphPanel.new('with alert conditions as an array', span=12)
                                 .addAlert('name of alert')
                                 .addConditions([{ c1: 'params' }, { c2: 'params' }]),
+  links: graphPanel.new('with links', span=12)
+          .addLink('foolinks')
+          .addLink('barlinks')
```

Comment:
@riconnon thanks for updating, but I'm afraid the whitespace still doesn't appease `jsonnet fmt`. The above two lines need to go left one space now. You can run this command locally to make the change:

```
docker run -w $PWD -v $PWD:$PWD --entrypoint sh sparkprime/jsonnet tests.sh update
```

It's the same command run on CircleCI with `update` appended. This indicates to `tests.sh` to update the files that don't match its formatting expectations.

Reply:
Thanks, that fixed it. I'd had difficulty earlier trying to get the jsonnet fmt tool to run locally.